test/integration/organisation_test.rb

```diff
@@ -396,18 +410,25 @@ class OrganisationTest < ActionDispatch::IntegrationTest
   it "shows the ministers for an organisation" do
     visit "/government/organisations/attorney-generals-office"
     assert page.has_css?(".gem-c-heading", text: "Our ministers")
-    assert page.has_css?('.gem-c-image-card .gem-c-image-card__title-link', text: 'Theresa May MP')
-    assert page.has_css?('.gem-c-image-card .gem-c-image-card__title-link', text: 'Stuart Andrew MP')
+    assert page.has_css?(".gem-c-image-card .gem-c-image-card__title-link", text: "Theresa May MP")
+    assert page.has_css?(".gem-c-image-card .gem-c-image-card__title-link", text: "Stuart Andrew MP")
   end
 
   it 'does not show the ministers section for no.10' do
     visit "/government/organisations/prime-ministers-office-10-downing-street"
-    refute page.has_css?('.gem-c-heading', text: 'Our ministers')
-    refute page.has_css?('.gem-c-image-card .gem-c-image-card__title-link', text: 'Theresa May MP')
+    refute page.has_css?(".gem-c-heading", text: "Our ministers")
+    refute page.has_css?(".gem-c-image-card .gem-c-image-card__title-link", text: "Theresa May MP")
   end
 
   it 'does not display ministers for organisations without minister data' do
     visit "/government/organisations/charity-commission"
-    refute page.has_css?('.gem-c-heading', text: 'Our ministers')
+    refute page.has_css?(".gem-c-heading", text: "Our ministers")
+  end
+
+  it 'displays the separate website page correctly' do
+    visit "/government/organisations/fire-service-college"
+    assert page.has_css?(".gem-c-organisation-logo")
+    assert page.has_css?(".gem-c-notice", text: "Fire Service College has a separate website")
```

Comment:
Once this has been implemented, we should check that we link to the organisation website too.